main.py

```diff
@@ -49,14 +63,22 @@ def reAward(fname):
 ###'''
 
 ##(/5) Task 3: CREATE code that will open a file and extract all the phone numbers 
+ #open file
+#loop through the file
+#extract the specific phone numbers regex
+#if the length of the extraction is not empty
+#print the phone number
 
 def rePhone(fname):
-  #open file
-  #loop through the file
-  #extract the specific phone numbers regex
-  #if the length of the extraction is not empty
-  #print the phone number
-  pass
+  fhand = open(fname,'r')
+  for line in fhand:
+    
+    extr = re.findall('902.*', line) # or 'PH: (.*)' if numbers are not only based in PEI
```

Comment:
Try using /d to capture digits or numbers. Look at the pattern of a phone number.